.github/workflows/package.yml

```diff
@@ -13,10 +15,6 @@ on:
     branches:
       - "main"
       - "develop"
-      - "integration"
-      - "releases/**"
```

Comment:
I guess we can keep this one in case we're pushing hotfixes to `releases/**` right before releasing it, as sometimes we need to just push late time stuff there without creating a PR (which I guess shouldn't be the case, but sometimes is)

Reply:
yes, makes sense